docs/source/requirements.rst

```diff
@@ -235,6 +235,8 @@ with *convention* requirements.
 ``docformatter`` currently provides these arguments for *style* requirements.
 ::
 
+    --black [boolean, default False]
```

Comment:
You'll need to merge the small changes from #168.  Also, look over the table of requirements and see if we need to add any around the new `--black` argument.

Reply:
Not sure what you mean by table of requirements? I merged in the change from #168

Reply:
Sorry, I wasn't clear, was I?  If you look at this [table](https://docformatter.readthedocs.io/en/latest/requirements.html) in the docs, you'll see a list of requirements.  Right now, there's nothing around `black` compatibility, but there should be and then there should be at least one test for each requirement to verify `docformatter` performs as expected.  After you look it over, let me know if you have any questions or think I'm out to lunch with the ask.

Reply:
I'll take care of updating the requirements table.

Reply:
Thanks - I had a look but was a bit confused so hadn't included it in my latest commit with the updated tests